tests/src/common/Wsk.scala

```diff
@@ -133,7 +133,7 @@ trait ListOrGetFromCollection extends FullyQualifiedNames {
         limit: Option[Int] = None,
         expectedExitCode: Int = SUCCESS_EXIT)(
             implicit wp: WskProps): RunResult = {
-        val params = Seq(noun, "list", resolve(namespace), "--auth", wp.authKey) ++
+        val params = Seq(noun, "list", resolve(namespace), "-i", "--auth", wp.authKey) ++
```

Comment:
-i | --insecure allows Go CLI to connect to controller in an insecure manner, ignoring certificate issues, such as self signed certs used by the local server.   The python CLI is hard coded to connect via insecure SSL.  For simple compatibility, we can add a -i option to the python CLI that's a noop - assuming the python CLI is truly being deprecated by the Go CLI... no need to maintain both CLIs, right?